README.md

```diff
@@ -166,6 +175,8 @@ ansible-playbook -i ./hosts\
 ansible-playbook -i ./hosts ./teardown.yaml
 ```
 
+> Note: this action teardowns only the services, the MAAS database is preserved in the `maas_postgres_*` nodes. You must remove these files manually before attempting to redeploy the stack.
```

Comment:
This is an issue with teardown that I was not aware of, by itself teardown is meant to restore the nodes to pre-install state. I think we should fix teardown (which needs updating anyway, with new changes to the playbooks) rather than tell people to do manual removal alongside running teardown?

Reply:
IMO this is an interesting feature as is (I don't like dropping databases without getting multiple confirmations first). What we need is a flag `purge_db` (or something like this) that the user needs to explicitly set if he wants to delete data files.